aggie-coding-club/Rev-Registration
# Add term select to navbar for schedule page

## Description

Pretty straightforward, this adds the SelectTerm to the navbar so it'll appear in the schedule page.

There's a few clean-up things I'd like to add (and also need to add some tests), but I wanted to get it out so we could discuss the look of it during the meeting.

## How to test

Select the term and make sure it navigates to the schedule page from the landing page. Also make sure it switches correctly when you're already on the schedule page.

## Screenshots

|Landing Page|Schedule Page|
|--|--|
|![chrome_3l5DEqoVju](https://user-images.githubusercontent.com/7295783/93025695-b8710b00-f5c5-11ea-9b3f-7c2954b38367.png)|![chrome_wMEyUwKNfF](https://user-images.githubusercontent.com/7295783/93025696-bb6bfb80-f5c5-11ea-9419-b3359aa9fbdf.png)|

## Related tasks

Closes #289
